docs/governance-and-automation/automated-workflows/orphaned-users-admin.md

```diff
@@ -6,22 +6,34 @@ description: This article explains how to activate the Orphaned Users policy.
 
 Syskit Point provides a predefined **Orphaned Users policy** that is applied tenant-wide and cannot be deleted. 
 
-This policy helps you detect and remove users who can potentially lead to risks and complications in managing workspaces. 
+This policy helps you identify and remove users who may pose risks or complications when managing workspaces.
 
-To edit the policy, navigate to the Policies settings screen and **click the Edit Policy (1)** icon.
+To edit the policy, go to the Policies settings screen and **click the Edit Policy icon**.
 
 ![Blocked Users with Assigned Licenses - Edit Policy](../../../static/img/orphaned-users-admin-edit-policy.png)
 
-The **Edit Policy** dialog opens where you can select the **Severity level (1)**.
-  * The severity level set by default for this policy is **Medium**
-  
-You can select between three levels of severity: 
+The **Edit Policy** dialog opens, where you can:
+* Select the **Severity level (1)**.
+   * The default severity level for this policy is **Medium**.
+   * You can choose from **three levels of severity**:
+     * High
+     * Medium
+     * Low
+* Choose how you want to **resolve the policy when a new vulnerability is detected**: 
+  * **Resolve Manually (2)** - selecting this option means the vulnerability is flagged on the Security & Compliance Dashboard, where you or other admins can take manual actions to resolve it.
+  * **Resolve Automatically (3)** - selecting this option means Syskit Point automatically removes orphaned users to resolve the vulnerability when it is detected.
+  * You can select which type of orphaned users should be automatically deleted by choosing one of three options:
+    * **Deleted Orphaned Users** - selecting this removes all deleted orphaned users from your workspaces.
+    * **Disabled Orphaned Users** - selecting this removes all disabled orphaned users from your workspaces.
+    * **All Orphaned Users** - selecting this removes both disabled and deleted orphaned users from your workspaces.
 
-  * High
-  * Medium
-  * Low
+Once you've selected the options that best suit your needs for this policy, **click the Save button (4)** to store your preferences.
 
-Once you've selected the one that best suits your needs for this policy, **click the Save button (2)** to store your preference. 
+![Blocked Users with Assigned Licenses - Severity](../../../static/img/orphaned-users-admin-policy.png)
 
+:::warning
 
-![Blocked Users with Assigned Licenses - Severity](../../../static/img/orphaned-users-admin-severity.png)
\ No newline at end of file
+**Please note!**  
+If automation is selected to resolve the Orphaned Users vulnerability, **Syskit Point also removes OneDrives** from sites with orphaned users.
```

Comment:
ako se ovo odnosi na 'Maknuti ODijevi iz detekcije za orphaned users policy' onda mi nije najjasnije
ali generalno bi maknuo ovaj please note. 
u sljedećem releaseu će se dodatno mijenjati taj detection pa možemo onda detaljno i točno napisati što se dešava
related item za sljedeći release #71860